
int8 UNKNOWN=-1
int8 DIGITAL=0
int8 ANALOG=1

int8 type
